hedera-node/hapi-utils/src/main/java/com/hedera/node/app/hapi/utils/ethereum/EthTxData.java

```diff
@@ -402,6 +402,10 @@ private static EthTxData populateEip1559EthTxData(RLPItem rlpItem, byte[] rawTx)
             return null;
         }
 
+        final var value = rlpList.get(6).asBytes().length == 0
+                ? BigInteger.ZERO
+                : new BigInteger(rlpList.get(6).asBytes());
```

Comment:
This is a better way to handle two's complement conversion. Before when we were passing a negative number(for example -1000) the result was 271474976710656. If there is another way to handle this, please let me know and I will change it.